actions/check-edrs.vbs

```diff
@@ -51,67 +51,67 @@ changelog = array()
+CALL MAXIS_case_number_finder(MAXIS_case_number)
+
+'changing footer dates to current month to avoid invalid months.
+MAXIS_footer_month = datepart("M", date)
+IF Len(MAXIS_footer_month) <> 2 THEN MAXIS_footer_month = "0" & MAXIS_footer_month
+MAXIS_footer_year = right(datepart("YYYY", date), 2)
 
 Dialog1 = ""
-BeginDialog Dialog1, 0, 0, 156, 80, "EDRS dialog"
-  EditBox 60, 10, 80, 15, MAXIS_case_number
+BeginDialog Dialog1, 0, 0, 201, 65, "ELECTRONIC DISQUALIFIED RECIPIENT SYSTEM "
+  EditBox 70, 5, 50, 15, MAXIS_case_number
+  EditBox 70, 25, 125, 15, worker_signature
   ButtonGroup ButtonPressed
-    OkButton 15, 55, 50, 15
-    CancelButton 80, 55, 50, 15
-  Text 5, 15, 50, 10, "Case Number:"
+    PushButton 130, 5, 65, 15, "ERDS TE02.08.127", POLI_TEMP_ERDS_button
+    OkButton 100, 45, 45, 15
+    CancelButton 150, 45, 45, 15
+  Text 5, 30, 60, 10, "Worker Signature:"
+  Text 5, 10, 50, 10, "Case Number:"
 EndDialog
 
-DO
-	dialog Dialog1
-	IF buttonpressed = 0 THEN stopscript
-	IF MAXIS_case_number = "" THEN MSGBOX "Please enter a case number"
-
-LOOP UNTIL MAXIS_case_number <> ""
-
-'Error proof functions
-Call check_for_MAXIS(False)
+Do
+    Do
+        err_msg = ""
+        DIALOG Dialog1  					'Calling a dialog without a assigned variable will call the most recently defined dialog
+		cancel_confirmation
+		IF MAXIS_case_number = "" OR (MAXIS_case_number <> "" AND len(MAXIS_case_number) > 8) OR (MAXIS_case_number <> "" AND IsNumeric(MAXIS_case_number) = False) THEN err_msg = err_msg & vbCr & "Please enter a valid case number."
+		IF ButtonPressed = POLI_TEMP_ERDS_button THEN CALL view_poli_temp("02", "08", "127", "") 'TE02.08.127 ELECTRONIC DISQUALIFIED RECIPIENT SYSTEM
+        If err_msg <> "" Then MsgBox "*** Resolve to Continue: " & vbNewLine & err_msg
+    Loop until err_msg = ""
+    Call check_for_password(are_we_passworded_out)
+Loop until are_we_passworded_out = FALSE
 
 'Creating a custom dialog for determining who the HH members are
 call HH_member_custom_dialog(HH_member_array)
 
-'Error proof functions
-Call check_for_MAXIS(False)
-
-'changing footer dates to current month to avoid invalid months.
-MAXIS_footer_month = datepart("M", date)
-IF Len(MAXIS_footer_month) <> 2 THEN MAXIS_footer_month = "0" & MAXIS_footer_month
-MAXIS_footer_year = right(datepart("YYYY", date), 2)
-
 Dim Member_Info_Array()
 Redim Member_Info_Array(UBound(HH_member_array), 4)
 
-
 'Navigate to stat/memb and check for ERRR message
 CALL navigate_to_MAXIS_screen("STAT", "MEMB")
 For i = 0 to Ubound(HH_member_array)
-
 	Member_Info_Array(i, 0) = HH_member_array(i)
-	'Navigating to selected memb panel
-	EMwritescreen HH_member_array(i), 20, 76
+	EMwritescreen HH_member_array(i), 20, 76 	'Navigating to selected memb panel
 	transmit
 
 	EMReadScreen no_MEMB, 13, 8, 22 'If this member does not exist, this will stop the script from continuing.
 	IF no_MEMB = "Arrival Date:" THEN script_end_procedure("This HH member does not exist.")
```

Comment:
Best practice: If the script is going to end, inform the user. This sounds like an err_msg they can correct. 

Add PRIV handling to MEMB navigation.